src/pages/Notes/Notes.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Navbar, Sidebar, NoteCard } from "../../components/index";
+import "./Notes.css";
+
+function Notes() {
+  return (
+    <div>
+      <Navbar />
+      <section className="d-flex">
+        <Sidebar />
+
+        <div className="notes-container">
+          <div className="note-inp-container mb-2">
+            <input
+              className="nav_searchBar add-note-inp"
+              type="text"
+              placeholder="Take a note..."
+            />
+            <button className="btn pri-btn-style">Filter</button>
+          </div>
+          <div>
+            <div>
+              <small className="font-size-small ml-1 f-weight-500">
+                PINNED
+              </small>
+              <div className="d-flex notecard-container">
+                <NoteCard />
+              </div>
+            </div>
+            <div>
+              <small className="font-size-small ml-1 f-weight-500">
```

Comment:
Here we can improve the code a little bit, not much. 
As the element structure for the `PINNED` and `OTHERS` are same almost, so you can loop over an array.
Something like,
```
[{_id: 001, value: "PINNED"}, {...other}].map(({_id, value}) => (
<div>
    <small>
       ... all others tag
))
```

See, if you found this interesting, you can refactor your code.

Reply:
Thanks I will look into it